lib/di/global/get_it_initializer.dart

```diff
@@ -345,6 +346,10 @@ class GetItInitializer {
         getIt.get<AppConfigLoader>(),
       ),
     );
+
+    getIt.registerSingleton<DownloadMediaFileInteractor>(
+      DownloadMediaFileInteractor(),
+    );
```

Comment:
why need it here?

Reply:
we still need an interactor, because when user close the image viewer, it will cancel the download